File: golb-p/client/app/model/Bookmark.js
Ext.define("Golb.model.Bookmark",
{
  extend : "Ext.data.Model",
  requires : [ "Ext.data.identifier.Uuid", "Ext.data.proxy.Direct" ],
  identifier : "uuid",
  fields : [ {
    name : "id",
    type : "string",
    allowNull : true,
    convert : null
  }, {
    name : "title",
    type : "string",
    validators : [ {
      type : "notBlank"
    } ]
  }, {
    name : "url",
    type : "string",
    validators : [ {
      type : "url"
    }, {
      type : "notBlank"
    } ]
  }, {
    name : "remark",
    type : "string"
  }, "tags" ],
  proxy : {
    type : "direct",
    api : {
      read : "bookmarkService.read",
      create : "bookmarkService.update",
      update : "bookmarkService.update",
      destroy : "bookmarkService.destroy"
    },
    reader : {
      rootProperty : "records"
    },
    writer : {
      writeAllFields : true
    }
  }
});

File: attic/tools/HTMLBookmark/src/ch/sr/bookmark/Bookmark.java
package ch.sr.bookmark;
public class Bookmark {
   
   private String title;
   private String url;
   
   public Bookmark(String title, String url) {
   	this.title = title;
   	this.url = url;
   }
   
   public String getTitle() {
   	return title;
   }
   
   public String getURL() {
   	return url;
   }
   
   public void setTitle(String title) {
   	this.title = title;
   }
   
   public void setURL(String url) {
   	this.url = url;
   }
   
}